src/ThreeDSAuth.res

```diff
@@ -0,0 +1,94 @@
+open Utils
+let getKeyValue = (json, str) => {
+  json
+  ->Js.Dict.get(str)
+  ->Belt.Option.getWithDefault(Js.Dict.empty()->Js.Json.object_)
+  ->Js.Json.decodeString
+  ->Belt.Option.getWithDefault("")
```

Comment:
Since we have integrated rescript core, the above code could be written as 
```
 json
  ->Dict.get(str)
  ->Option.getOr(Dict.make()->JSON.Encode.object)
  ->JSON.Decode.string
  ->Option.getOr("")
  ```
  
  We can stop using `Belt.`, `Js.`, etc in our code.
  
  This change doesn't have to be done in this PR itself, but something to keep in mind going forward.